core/lib/prover_interface/src/api.rs

```diff
@@ -18,10 +17,9 @@ use crate::{
 #[derive(Debug, Serialize, Deserialize)]
 pub struct ProofGenerationData {
     pub l1_batch_number: L1BatchNumber,
-    pub data: PrepareBasicCircuitsJob,
+    pub witness_input_data: WitnessInputData,
     pub protocol_version: ProtocolSemanticVersion,
     pub l1_verifier_config: L1VerifierConfig,
-    pub eip_4844_blobs: Eip4844Blobs,
```

Comment:
nope, in gateway we are still saving merkle paths data in bucket along with WitnessInputData - this way when we will rollback BWG, it will still get correct data, so there is no need to rollback data handler and gateway